contracts/external/dao-migrator/src/testing/test_migration.rs

```diff
@@ -0,0 +1,147 @@
+use std::borrow::BorrowMut;
+
+use dao_core::query::SubDao;
+
+use crate::{
+    testing::{helpers::ExecuteParams, state_helpers::query_state_v2},
+    ContractError,
+};
+
+use super::{
+    helpers::VotingType,
+    setup::{execute_migration, setup_dao_v1},
+    state_helpers::query_state_v1,
+};
+
+#[test]
+fn test_execute_migration() {
+    let (mut app, module_addrs, v1_code_ids) = setup_dao_v1(VotingType::Cw20);
+
+    let mut test_state_v1 = query_state_v1(
+        &mut app,
+        module_addrs.proposal.clone(),
+        module_addrs.voting.clone(),
+    );
+    //NOTE: We add 1 to count because we create a new proposal in execute_migration
+    test_state_v1.proposal_count += 1;
+
+    execute_migration(app.borrow_mut(), &module_addrs, v1_code_ids, None).unwrap();
```

Comment:
could we also query and assert after this migration that the module is no longer one of the proposal modules?